.github/workflows/pr-auto.yml

```
@@ -17,16 +17,25 @@ jobs:
-               github.event.pull_request.title.startsWith('[refactor]') && 'refactor' }}
+        with: { labels: bug }
+
+      - name: 'chore' 라벨 추가
+        if: startsWith(github.event.pull_request.title, 'Chore') || startsWith(github.event.pull_request.title, 'chore')
+        uses: actions-ecosystem/action-add-labels@v1
+        with: { labels: chore }
+
+      - name: 'feature' 라벨 추가
+        if: startsWith(github.event.pull_request.title, 'Feat') || startsWith(github.event.pull_request.title, 'feat')
+        uses: actions-ecosystem/action-add-labels@v1
+        with: { labels: feature }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)